app/policies/profile/password_policy.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class Profile::PasswordPolicy < ApplicationPolicy
```

Comment:
instead of creating a policy for Profile::PasswordPolicy here, you could integrate this into User policy since this action is user related. action would be update_password
maybe check resetpassword action in User Human Policy